confgenerator/fluentbit/service.go

```diff
@@ -31,12 +31,6 @@ func (s Service) Component() Component {
 			// Log_File is set by Fluent Bit systemd unit (e.g. /var/log/google-cloud-ops-agent/subagents/logging-module.log).
 			"Log_Level": s.LogLevel,
 
-			// https://docs.fluentbit.io/manual/administration/monitoring
-			// Enable a built-in HTTP server that can be used to query internal information and monitor metrics of each running plugin.
-			"HTTP_Server": "On",
```

Comment:
The answer is probably yes. But just checking: Is this `Off` by default?

Reply:
Yes, I double checked and the behavior of ommiting and setting to "Off" is the same.

Reply:
Great. Thanks for checking!